backend/src/Edna/Dashboard/DB/Query.hs

```diff
@@ -164,6 +163,9 @@ getExperiments mProj mComp mTarget sorting pagination =
         )
       )
   where
+    eExperimentId' :: ExperimentRec -> SqlSerial Word32
+    eExperimentId' = eExperimentId
```

Comment:
Can you please double check that it's actually needed in the final version of code?

Reply:
well, now it compiles without it))